wiki/models/__init__.py

```diff
@@ -42,13 +42,21 @@
     raise ImproperlyConfigured(
         'django-wiki: needs django.contrib.sites in INSTALLED_APPS')
 
+TEMPLATE_CONTEXT_PROCESSORS = []
+if hasattr(django_settings, 'TEMPLATES'):
+    # Django 1.8 compat
+    backends = [b for b in django_settings.TEMPLATES if b.get('BACKEND', '') == 'django.template.backends.django.DjangoTemplates']
+    if len(backends) == 1:
+        TEMPLATE_CONTEXT_PROCESSORS = backends[0].get('OPTIONS', {}).get('context_processors', [])
+
 if not 'django.contrib.auth.context_processors.auth' in django_settings.TEMPLATE_CONTEXT_PROCESSORS:
```

Comment:
You're right, my change doesn't work correctly for older projects. I'll submit a fix.